scripts/node/icons/helpers/cleanExports.js

```diff
@@ -65,4 +64,7 @@ const cleanExports = (ext) => {
   });
 };
 
-module.exports = cleanExports;
+module.exports = {
+  cleanExports,
+  normalizePathToFile,
+};
```

Comment:
I'm not sure with `module.exports` how to do both named exports and default exports, but it doesn't really matter anyway.

Reply:
i'm not 100% sure, but i think you'd do 
```
module.exports = {
  cleanExports,
  normalizePathToFile,
  default: /* module here */
};
```

Reply:
unfortunately doesn't work for me.